RELEASE_NOTES.md

```diff
@@ -2,8 +2,16 @@
 
 ## Summary
 
-This is the initial release, extracted from the [SDK v1.0.0rc601](https://github.com/frequenz-floss/frequenz-sdk-python/releases/tag/v1.0.0-rc601).
```

Comment:
I wouldn't remove this until we release v1.0.0, we can add that it is based on that code but with some updates.